### PR TITLE
Port test suite source map test.

### PR DESCRIPTION
**Pull request by nicolasstucki**

### CODE REVIEW

**Comment by nicolasstucki** [comment]
Review by @sjrd.

File: project/Build.scala [hunk 1]
```diff
@@ -1248,7 +1248,10 @@ object Build extends sbt.Build {
           })
 
           val outFile = dir / "SourceMapTest.scala"
-          IO.write(outFile, replaced.replace("0/*<testCount>*/", i.toString))
+          val unitTests =
+            (0 until i).map(i => s"@Test def workTest$i(): Unit = test($i)").mkString("; ")
+          IO.write(outFile,
+            replaced.replace("@Test def workTest(): Unit = sys.error(\"stubs\")", unitTests))
```
Comment (sjrd) [inline project/Build.scala]: 4 spaces

**Comment by sjrd** [comment]
Otherwise the "Port testSuite SourceMapTest" commit LGTM. I would like the other commits to be on their own PRs.

**Comment by sjrd** [comment]
LGTM

**Pull request merged**